rule/error_strings.go

```diff
@@ -178,21 +178,29 @@ func (lintErrorStrings) checkArg(expr *ast.CallExpr, arg int) (s *ast.BasicLit,
 func lintErrorString(s string) (isClean bool, conf float64) {
 	const basicConfidence = 0.8
 	const capConfidence = basicConfidence - 0.2
-	first, firstN := utf8.DecodeRuneInString(s)
+
 	last, _ := utf8.DecodeLastRuneInString(s)
 	if last == '.' || last == ':' || last == '!' || last == '\n' {
 		return false, basicConfidence
 	}
-	if unicode.IsUpper(first) {
-		// People use proper nouns and exported Go identifiers in error strings,
-		// so decrease the confidence of warnings for capitalization.
-		if len(s) <= firstN {
-			return false, capConfidence
+
+	first, firstN := utf8.DecodeRuneInString(s)
+	if !unicode.IsUpper(first) {
+		return true, 0
+	}
+
+	// People use proper nouns and exported Go identifiers in error strings,
+	// so decrease the confidence of warnings for capitalization.
+	for _, r := range s[firstN:] {
+		if unicode.IsSpace(r) {
+			break
 		}
```

Comment:
Here I'm coming with hypothetical edge cases:
- errors.New("")
- errors.New("A")
- errors.New("KO")
- errors.New("U.S.A.")
- errors.New("SCREAMING_SNAKE_CASE")

I'm not even sure if they should be considered as valid, but I'm raising them as counter examples.

Also, non-Latin language could be tested, they work right now, for records.

Reply:
I didn't get if these edge cases should be considered okay or rise a failure. It's clear to me that `errors.New("")` is not okay and we could raise a failure but what to say about the other examples? 

About non-Latin language(s):
1. there are [plenty of non-latin languages](https://sites.google.com/site/worldfactsinc/Non-Latin-Script-Languages-Of-The-World) and... 
2. ...IMO the rule "error messages must not start with a capital letter" is a Latin-centric rule, not applicable to many of the non-Latin languages.

Reply:
I'm fine with this 👍

The use cases you added to test files are enough